src/file_formats.jl

```diff
@@ -0,0 +1,134 @@
+#  Copyright 2017, Iain Dunning, Joey Huchette, Miles Lubin, and contributors
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+const MOF = MathOptFormat
```

Comment:
I'm still uncomfortable with exporting other modules (https://github.com/JuliaOpt/JuMP.jl/issues/1789). Do we need to export MOF or can we just re-export the two enums used here?